api/apps/geoprocessing/src/migrations/geoprocessing/1692254627921-AddsCostSurfacePuDataGeoDb.ts

```diff
@@ -9,9 +9,14 @@ export class AddsCostSurfacePuDataGeoDb1692254627921
     await queryRunner.query(`
       CREATE TABLE cost_surface_pu_data (
         id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+        -- consider marking this as not null, as it is for projects, and maybe required for the GC
         cost_surface_id uuid NOT NULL,
-        puid uuid UNIQUE NOT NULL REFERENCES projects_pu(id) ON DELETE CASCADE,
-        cost float8 NOT NULL
+        puid uuid NOT NULL REFERENCES projects_pu(id) ON DELETE CASCADE,
+        cost float8 NOT NULL,
+        -- modifying this as I need the change for tests
+        UNIQUE (cost_surface_id, puid)
```

Comment:
The problem is that this migration file was previously approved (and therefore executed everywhere). I have already spoken about that with @KevSanchez, so that either he will modify the constraint in his PR or we will push the change on a minimal PR and remove this change here.

I had to do temporarily do this so my tests reflect what is going to be reality, since all migrations are run in CI. But IRL my PR wouldn't work, we need a new migration reflecting the change.

Let's clarify this

Reply:
thanks, as we discusse we'll silently pretend that `AddCostSurdaceApiDb1691980238543` never happened in local dev envs and on m23

btw since you're updating this migration, consider to quick-fix the name of the class and the file (`Surdace -> Surface`) - I only noticed this now (ok to keep it as is, ofc)